pom.xml

```diff
@@ -27,7 +27,7 @@
 		<cloud.sdk.version>5.10.0</cloud.sdk.version>
 		<xsuaa.version>3.5.2</xsuaa.version>
 		<cf-java-logging-support.version>3.8.3</cf-java-logging-support.version>
-		<cds.install-cdsdk.version>8.0.2</cds.install-cdsdk.version>
+		<cds.cdsdk-version>8.0.2</cds.cdsdk-version>
```

Comment:
For my understanding as I don´t find usage of this property: is it meant to override the one that is defined in `cds-services`?

Reply:
Hi, this property is used by the goal `cds`: https://cap.cloud.sap/docs/java/assets/cds-maven-plugin-site/cds-mojo.html#cdsdkVersion
If this property is set. the goal `cds` uses npx to install the package @sap/cds-dk. So we don't need an explicit execution of goal `install-cdsdk`.